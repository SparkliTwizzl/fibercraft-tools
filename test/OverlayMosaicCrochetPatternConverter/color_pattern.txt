red	10sc
blue	5sc,5dc
green	10dc
yellow	4sc,6dc
purple	10sc
orange	3dc,7sc
pink	10dc
brown	6sc,4dc
black	10sc
white	2dc,8sc
