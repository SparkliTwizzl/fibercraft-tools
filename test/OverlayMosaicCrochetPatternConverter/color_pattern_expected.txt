    1. red        bs, [1] 10sc, bs
    2. blue       bs, [1] 5sc, [6] 5dc, bs
    3. green      bs, [1] 10dc, bs
    4. yellow     bs, [1] 4sc, [5] 6dc, bs
    5. purple     bs, [1] 10sc, bs
    6. orange     bs, [1] 3dc, [4] 7sc, bs
    7. pink       bs, [1] 10dc, bs
    8. brown      bs, [1] 6sc, [7] 4dc, bs
    9. black      bs, [1] 10sc, bs
   10. white      bs, [1] 2dc, [3] 8sc, bs
